view.rb

```diff
@@ -0,0 +1,17 @@
+  class View
```

Comment:
Your view should be handling _all_ of your output to the user. There should be no puts statements anywhere else.